packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -1550,19 +1550,13 @@ space
         })
       })
 
-      it('sets type to parent when subject isnt element', () => {
```

Comment:
This is a behavioral change: .contains() now logs as a child whenever it's used as a child command. Previously, it logged as a child if used as a child command *unless the previous command was body/window*.

This exception is not a useful one. Simple, easy to understand rules are preferable.